### ISSUE TITLE
Could not build module 'Firebase'

### ISSUE THREAD

**Issue by mesqueeb** [dpa99c/cordova-plugin-firebasex#206]
# Bug report

**Current behavior:**

I'm able to open a dev server via cordova with a simulator.
I'm also able to build the app which says `** BUILD SUCCEEDED **`

However, if I open XCode and try `product > archive` I get this error:

<img width="697" alt="image" src="https://user-images.githubusercontent.com/3253920/68107175-54060980-ff27-11e9-9e48-945df1893461.png">

This also happens when I try to build directly to my iPhone via XCode or via cordova command line.

The strange thing is that the project WORKS without problem via the cordova dev server and I even get notifications.

**Screenshots**

**Environment information**

- Cordova CLI version 
	- `cordova -v`
        - > 9.0.0 (cordova-lib@9.0.1)
- Cordova platform version
	- `cordova platform ls`
        - > android 8.1.0
        - > ios 5.0.1
- Plugins & versions installed in project (including this plugin)
    - `cordova plugin ls`

```
cordova-android-firebase-gradle-release 4.0.0 "cordova-android-firebase-gradle-release"
cordova-android-play-services-gradle-release 4.0.0 "cordova-android-play-services-gradle-release"
cordova-plugin-androidx 1.0.2 "cordova-plugin-androidx"
cordova-plugin-androidx-adapter 1.1.0 "cordova-plugin-androidx-adapter"
cordova-plugin-firebasex 6.1.0 "Google Firebase Plugin"
cordova-plugin-googleplus 8.2.0 "Google SignIn"
cordova-plugin-inappbrowser 3.1.0 "InAppBrowser"
cordova-plugin-ionic-webview 4.1.3 "cordova-plugin-ionic-webview"
cordova-plugin-splashscreen 5.0.3 "Splashscreen"
cordova-plugin-whitelist 1.3.4 "Whitelist"
cordova-plugin-x-socialsharing 5.6.2 "SocialSharing"
cordova-support-google-services 1.3.2 "cordova-support-google-services"
es6-promise-plugin 4.2.2 "Promise"
```

- Dev machine OS and version, e.g.
    - OSX
        - `sw_vers`
```
ProductName:	Mac OS X
ProductVersion:	10.15.1
BuildVersion:	19B88
```

_iOS build issue:_
- Node JS version
    - `node -v`
    - > v12.12.0
- XCode version
> Version 11.1 (11A1027)

**Comment by dpa99c** [comment]
This error indicates an issue with the Cocoapods setup in your dev environment such that the Firebase SDK dependency was not satisfied during the build process. Please see the [Cocoapods section](https://github.com/dpa99c/cordova-plugin-firebasex#cocoapods) in the documentation.
Also try building and running the [example project](https://github.com/dpa99c/cordova-plugin-firebasex-test) as a known working reference to validate your dev environment.

**Comment by mesqueeb** [comment]
@dpa99c Thank you very much for your kind advice.
It worked when I followed the exact documentation concerning the cocoapods, and then opened & archived the project via `.xcworkspace`. Does this mean I'll always need to archive and upload via `xcworkspace` going forward?

This was a change for me because I came from `cordova-plugin-firebase` and switched to this one. ;)

thanks a lot for all your work on the fork!
